src/pages/signin/SignInPage.js

```diff
@@ -96,24 +137,33 @@ function SignInPage({credentials, account, isInModal, activeClients}) {
      * and we need it here since welcome text(`welcomeText`) also depends on it */
     const [isUsingRecoveryCode, setIsUsingRecoveryCode] = useState(false);
 
-    useEffect(() => Performance.measureTTI(), []);
-    useEffect(() => {
-        App.setLocale(Localize.getDevicePreferredLocale());
-    }, []);
```

Comment:
Any idea, why this was removed?

Reply:
🤔 i didn't intentionally remove it. Lemme see if it's on main now and if so I'll add it back in

Reply:
it's on main, added it back!